boards/arm/nrf51_pca10028/doc/nrf51_pca10028.rst

```diff
@@ -45,21 +45,33 @@ hardware features:
 +-----------+------------+----------------------+
 | Interface | Controller | Driver/Component     |
 +===========+============+======================+
-| NVIC      | on-chip    | nested vectored      |
-|           |            | interrupt controller |
 +-----------+------------+----------------------+
-| RTC       | on-chip    | system clock         |
-+-----------+------------+----------------------+
-| UART      | on-chip    | serial port          |
+| ADC       | on-chip    | adc                  |
 +-----------+------------+----------------------+
 | GPIO      | on-chip    | gpio                 |
 +-----------+------------+----------------------+
+| I2C       | on-chip    | i2c                  |
++-----------+------------+----------------------+
 | FLASH     | on-chip    | flash                |
 +-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
 | RADIO     | on-chip    | Bluetooth            |
 +-----------+------------+----------------------+
+| SPI       | on-chip    | spi                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
 | RTT       | on-chip    | console              |
 +-----------+------------+----------------------+
+| TIMER     | on-chip    | clock_control        |
```

Comment:
The clock_control driver does not seem to provide a timer functionality.

Reply:
It does, for starting and stopping it.

Reply:
oh right sorry. Not the TIMER, just the clock itself! will fix.